content/en/building/training/training-cards-resources/_index.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: "Training Cards Resources"
 linkTitle: "Training Cards Resources"
-weight: 15
+weight: 3
 description: >
   Training cards for CHT
 keyword: training
 relatedContent: >
-  building/features/training
+  building/training/training-cards
```

Comment:
This line is duplicated. I believe it should be removed?

Reply:
Done!